WordPressAuthenticator/Unified Auth/View Related/Password/PasswordViewController.swift

```diff
@@ -373,6 +402,59 @@ private extension PasswordViewController {
 
         UIAccessibility.post(notification: .screenChanged, argument: passwordField)
     }
+
+    /// Makes the call to request a magic authentication link be emailed to the user.
+    ///
+    func requestAuthenticationLink() {
+        loginFields.meta.emailMagicLinkSource = .login
+
+        let email = loginFields.username
+        guard email.isValidEmail() else {
+            // This is a bit of paranoia as in practice it should never happen.
```

Comment:
I think it's totally fine to cover this case even if it might be an edge case. An option to make it more readable maybe is to extract the alert part in a separate method or computed variable?